src/lib/features/artifacts.js

```diff
@@ -139,8 +139,17 @@ export default class Artifacts {
       };
 
       try {
-        let digest = await uploadToS3(taskHandler.queue, taskId, runId, stream,
+        let start = Date.now();
+        let {digest, size} = await uploadToS3(taskHandler.queue, taskId, runId, stream,
                          entryName, expiry, headers);
+
+        let monitor = taskHandler.runtime.monitor;
+        let elapsedTime = (Date.now() - start) / 1000; // in seconds
+
+        monitor.measure('artifacts.size', size);
+        monitor.measure('artifacts.uploadTime', elapsedTime);
+        monitor.measure('artifacts.throughput', size / elapsedTime); // Bytes/sec
```

Comment:
is it possible to have this as a per-region and also a global metric?  Since we're aggregating the data points *before* sending them to SignalFX, the only chance we have to tell the difference between global and per-region is if we note it before sending to statsum here.